clingo/tests/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -euo pipefail
+
+TESTDIR="$(dirname "${0}")"
+
+if [ -z "${1:-}" ]; then
+  echo "Usage: $0 FULLY_QUALIFIED_PACKAGE_IDENT"
+  exit 1
+fi
+
+TEST_PKG_IDENT="$1"
+
+hab pkg install --binlink core/bats
```

Comment:
This should be `hab pkg install core/bats --binlink` as of Habitat 0.81.0

Reply:
Updated and also [raised PR against design document](https://github.com/habitat-sh/core-plans/pull/2585) so that future test cases follow this rule.